daffodil-cli/src/it/scala/org/apache/daffodil/saving/TestCLISaveParser.scala

```diff
@@ -189,11 +189,13 @@ class TestCLISaveParser {
       shell.sendLine(String.format("%s -vv save-parser -s %s -r matrix %s", Util.binPath, testSchemaFile, savedParserFile.getName()))
       shell.expect(contains("[compile]"))
 
-      shell.sendLine(String.format("%s -vvv save-parser -s %s -r matrix %s", Util.binPath, testSchemaFile, savedParserFile.getName()))
-      shell.expect(contains("[debug]"))
+      // Too much output
+      // shell.sendLine(String.format("%s -vvv save-parser -s %s -r matrix %s", Util.binPath, testSchemaFile, savedParserFile.getName()))
+      // shell.expect(contains("[debug]"))
 
-      shell.sendLine(String.format("%s -vvvv save-parser -s %s -r matrix %s", Util.binPath, testSchemaFile, savedParserFile.getName()))
-      shell.expect(contains("[oolagdebug]"))
+      // Too much output
+      // shell.sendLine(String.format("%s -vvvv save-parser -s %s -r matrix %s", Util.binPath, testSchemaFile, savedParserFile.getName()))
+      // shell.expect(contains("[oolagdebug]"))
```

Comment:
Any chance this could be split into a separate pull request? I think we have a bug for removing the excessive test output.

Reply:
I can do that.